__tests__/endpoints.test.js

```diff
@@ -75,6 +75,24 @@ describe('NC news endpoint tests',()=>{
                 })
             })
         })
-        
+        describe('GETting all articles',()=>{
+            test('/api/articles',()=>{
+                return request(app)
+                .get('/api/articles')
+                .expect(200)
+                .then(({body})=>{
+                    body.articles.forEach(article=>{
```

Comment:
There is an assertion that needs to be here 👀 If the body.articles is an empty array, the assertions in the .forEach won't run andthe test will automatically pass 🫨 So we need an assertion to check that the array has a length property.